src/makeCalculator.js

```diff
@@ -38,6 +38,41 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+
+    result: 0,
```

Comment:
Don't add redundant spaces
```suggestion
  const calculator = {
    result: 0,
```